bin/publish-swagger-docs.sh

```diff
@@ -3,6 +3,7 @@
 # central reform-api-docs repo. Updates reform-api-docs spec if needed
 
 # assign environment variables
+# shellcheck disable=SC1091
```

Comment:
what does it do?

Reply:
added a note in description mate 😉 . makes shellcheck valid without any issues

Reply:
I'm blind :P